frontend/catalyst/qjit_device.py

```diff
@@ -160,19 +167,19 @@ def __init__(
         self.backend_kwargs = backend.kwargs if backend else {}
         device_name = backend.device_name if backend else "default"
 
-        shots_present = shots is not None
-        self._operations = get_qjit_pennylane_operations(target_config, shots_present, device_name)
-        self._observables = get_pennylane_observables(target_config, shots_present, device_name)
+        program_features = ProgramFeatures(shots is not None)
+        target_device_config = get_device_config(target_config, program_features, device_name)
+        self.caps = get_qjit_device_config(target_device_config)
```

Comment:
Could rename it something like properties? Not sure to understand why it is named caps

Reply:
What do you think about "capabilities" (the full word)? I think "properties" is way too abstract.

Reply:
It is better :+1: